charts/jenkins/jobs/nuxeo/lts/test-nuxeo-server-against-es7.17.groovy.gotmpl

```diff
@@ -0,0 +1,52 @@
+pipelineJob('nuxeo/lts/test-nuxeo-server-against-es7.17') {
+  // Description
+  description('Test Nuxeo against elasticsearch 7.17')
+  definition {
+    // Definition
+    cpsScm {
+      scm {
+        // SCM
+        git {
+          // Repositories
+          remote {
+            // Repository URL
+            url('https://github.com/nuxeo/nuxeo-lts')
+            // Credentials
+            credentials('github-branch-source')
+          }
+          // Branches to build
+          branches('^${NUXEO_BRANCH}')
+          browser {
+            // Repository browser
+            githubWeb {
+              // URL
+              repoUrl('https://github.com/nuxeo/nuxeo-lts')
+            }
+          }
+        }
+      }
+      // no lightweight checkout in order to make branch parameter work
+      lightweight(false)
+      // Script Path
+      scriptPath('ci/Jenkinsfiles/build-es7.17-compat.groovy')
+    }
+  }
+  parameters {
+    string {
+      name('NUXEO_BRANCH')
+      description('Branch to release.')
+      trim(true)
+    }
+    string {
+      name('NUXEO_BUILD_VERSION')
+      defaultValue('')
+      description('Version of the promoted Nuxeo Server build.')
```

Comment:
```suggestion
      description('Version of the Nuxeo Server build to test.')
```
Indeed, though this pipeline will be triggered by the release orchestrator, it can be launched manually at any time to validate a given build of a given branch. So, it's not strictly tied to a release.